packages/web/src/pages/upload-page/fields/SourceFilesField.tsx

```diff
@@ -113,6 +113,9 @@ export const SourceFilesField = () => {
     const stemsCategories =
       stemsValue?.map((stem) => stemCategoryFriendlyNames[stem.category]) ?? []
     values = [...values, ...stemsCategories]
+
+    if (values.length === 0) return null
```

Comment:
fixes issue where the preview would render for empty array, causing the field to have extra padding at the bottom